compiler/backend/src/org/jetbrains/kotlin/codegen/inline/RemappingClassBuilder.java

```diff
@@ -121,7 +121,7 @@ public MethodVisitor newMethod(
     @Override
     @NotNull
     public AnnotationVisitor newAnnotation(@NotNull String desc, boolean visible) {
-        return new AnnotationRemapper(builder.newAnnotation(remapper.mapDesc(desc), visible), remapper);
```

Comment:
Otherwise, the build fails because warnings are treated as errors. The old API is deprecated but still exists, preserving binary compatibility.